### ISSUE TITLE
It is impossible to export properly a PriceBookEntry standard object

### ISSUE THREAD

**Issue by MyMarvel** [texei/texei-sfdx-plugin#52]
When you try to export a standard PricebookEntry object like this:
`sfdx texei:data:export --objects Pricebook2,Product2,PricebookEntry --outputdir ./data`
there are no references in the generated PricebookEntry.json either to Product2 neither to Pricebook2 records. It contains just the following information, which is useless without those references:
`    {
      "attributes": {
        "type": "PricebookEntry",
        "referenceId": "PriceBookEntryRef1"
      },
      "UnitPrice": 25000,
      "IsActive": true,
      "UseStandardPrice": false
    },`
Haven't you ever use this standard object in your work?
I think it is a bug and should be fixed. You can give me guidance about how to fix that or you can fix it by yourself. Thanks a lot.
In case you need more information: whenever you try to import the generated data into a scratch org, you are getting, obviously, an error like
`Error importing record PriceBookEntryRef34: REQUIRED_FIELD_MISSING-Required fields are missing: [Pricebook2Id, Product2Id](Pricebook2Id,Product2Id)`
which is obvious since we do not have such references in the original data file.

**Comment by FabienTaillon** [comment]
Hi @MyMarvel, yes this use case is a little bit more difficult to handle because:
* All Products having a Pricebook Entry linked to any custom Pricebook must also have a Pricebook Entry linked to the Standard Pricebook (so you must import Pricebook Entries linked to the Standard Pricebook first)
* Standard Pricebook is part of the org, it can’t be created or replaced by any other Pricebook

I've blogged about it [here](https://blog.texei.com/easily-load-data-into-your-scratch-org-or-sandbox-part-2-c00604462a97), just scroll to the "One specific word: Standard Pricebook" section.

Basically you'll have to use a data plan file instead of just running the command. You can create a template file via the `sfdx texei:data:plan:generate --objects Pricebook2,Product2,PricebookEntry --outputdir ./data` command and then update the filters accordingly.

In the blog post there is an example about what you're trying to achieve, let me know if it's unclear.

**Comment by FabienTaillon** [comment]
@MyMarvel you're using Standard Price Book, right ?

**Comment by jsardello** [comment]
For anyone who might be digging around trying to set this up themselves, be aware that exporting the Standard Pricebook generates a slightly different file. 

Pricebook2.json looks like this:
`{
  "records": [
    {
      "attributes": {
        "type": "Pricebook2",
        "referenceId": "StandardPriceBook"
      },
      "Name": "Standard Price Book",
      "IsActive": false,
      "Description": null
    }
  ]
}`

Notice that the referenceId is "StandardPriceBook" instead of the normal "PricebookRef1"

So if you wanted to create a PriceBookEntry based on the Standard Price Book, you need this in your data-plan.json:
` {
      "name": "Pricebook2",
      "label": ""
    },
    {
      "name": "PricebookEntry",
      "label": "Standard Pricebook",
      "filters": "Pricebook2.IsStandard = true"
    }`

And your PricebookEntry.json file looks something like this:
`{
  "records": [
    {
      "attributes": {
        "type": "PricebookEntry",
        "referenceId": "PricebookEntryRef1"
      },
      "UnitPrice": 100,
      "IsActive": true,
      "Pricebook2Id": "StandardPriceBook",
      "Product2Id": "ProductRef1",
      "UseStandardPrice": false
    }
  ]
}`
